core/esmf-aspect-model-java-generator/src/test/java/org/eclipse/esmf/aspectmodel/java/AspectModelJavaGeneratorTest.java

```diff
@@ -1109,4 +1122,18 @@ public void testGenerateAspectWithMultipleInheritance() throws IOException {
             List.of( "@JsonTypeInfo(use = JsonTypeInfo.Id.NAME)",
                   "@JsonSubTypes({ @JsonSubTypes.Type(value = TestEntity.class, name = \"TestEntity\") })" ) );
    }
+
+   @Test
+   void testGenerateAspectWithPrefixAndPostfix() throws IOException {
+      final ImmutableMap<String, Object> expectedFieldsForAspectClass = ImmutableMap.<String, Object> builder()
+            .put( "testProperty", String.class )
+            .build();
+
+      final TestAspect aspect = TestAspect.ASPECT_WITH_PROPERTY;
+      final GenerationResult result = TestContext.generateAspectCode()
+            .apply( getGenerators( aspect, "Base", "Postfix" ) );
+      result.assertNumberOfFiles( 1 );
+      result.assertFields( "BaseAspectWithPropertyPostfix", expectedFieldsForAspectClass, new HashMap<>() );
+      assertConstructor( result, "BaseAspectWithPropertyPostfix", expectedFieldsForAspectClass );
+   }
```

Comment:
Please also add one more test for TestAspect.ASPECT_WITH_ENTITY and check that the prefix/postfix is also applied to the class that is generated for the Entity